### PR TITLE
Add doc pages for new formats in 5.1.0

### PR DESCRIPTION
**Pull request by melissalinkert**
New readers mentioned in gh-1645 are now documented.  Builds should be green, and there shouldn't be anything obviously weird in the new pages.

### CODE REVIEW

**Comment by hflynn** [comment]
Could you add an update to the first line of https://www.openmicroscopy.org/site/support/bio-formats5/formats/index.html to indicate over 140 formats now? Seems a bit daft me opening a separate PR to change one number!

**Comment by joshmoore** [comment]
/me can hear @sbesson thinking of a way to autogenerate it now....

**Comment by melissalinkert** [comment]
Commit pushed to update all of the places I could find where "over XXX" formats are mentioned.

File: components/autogen/src/format-pages.txt [hunk 1]
```diff
@@ -79,6 +79,18 @@ notes = Known deficiencies: \n
...
 - Texture data is currently ignored.
 
+[Amnis FlowSight]
+extensions = .cif
+owner = `Amnis <http://www.amnis.com/>`_
+bsd = yes
+export = no
+weHave = * a few sample datasets
+pixelsRating = Good
+opennessRating = Good
```
Comment (sbesson) [inline components/autogen/src/format-pages.txt]: `metadataRating` is missing from this dictionary

**Comment by melissalinkert** [comment]
Commit pushed to add the missing ```metadataRating```.

**Comment by hflynn** [comment]
All looks fine to me, thanks.

**Comment by rleigh-dundee** [comment]
Looks fine to me also.

**Comment by melissalinkert** [comment]
--no-rebase

**Pull request merged**